packages/create-cloudflare/src/pages.ts

```diff
@@ -61,7 +66,7 @@ export const runPagesGenerator = async (args: C3Args) => {
 	}
 	await updatePackageScripts(ctx);
 	await offerGit(ctx);
-	await gitCommit(ctx);
+	await gitCommit(ctx, generatePagesCommitMessage(ctx));
```

Comment:
For consistency and to accommodate some changes I'd like to make later, it would be better if we kept the signature as is, and did some branching logic inside `gitCommit` function itself to generate the commit one way or the other depending on if it's pages or workers.

You can check this condition by evaluating the truthiness of `ctx.framework` -- it will be defined for pages apps but not for workers. Ideally, we'd have a dedicated field that gets set early on in the process (i.e. `ctx.projectType = 'workers'`) which we could key off of, but we can use `ctx.framework` as a close enough proxy for now.

Reply:
ah ok thanks, I'll move the change in the function sorry about that 🙂:+1: